Undefined constant Cookie in nightly

Lucky is now getting this error on Crystal nightly

```
 Showing last frame. Use --error-trace for full trace.

In /home/runner/work/_temp/crystal-nightly-true-undefined/share/crystal/src/http/cookies.cr:9:24

 9 | include Enumerable(Cookie)
                        ^-----
Error: Error: undefined constant Cookie

Did you mean 'Cookies'?
Error: Process completed with exit code 1.
```

https://github.com/crystal-lang/crystal/blame/28479a405a6d48590d37cf4ede8523edebbaccc9/src/http/cookies.cr#L9

```
crystal --version
Crystal 1.16.0-dev [2a0313342] (2025-03-29)
```

--- a/src/http/cookie.cr
+++ b/src/http/cookie.cr
@@ -1,5 +1,3 @@
-require "./common"
-
 module HTTP
   # Represents a cookie with all its attributes. Provides convenient access and modification of them.
   #
@@ -352,3 +350,5 @@ module HTTP
     end
   end
 end
+
+require "./common"

